module/config.mjs

```diff
@@ -2616,7 +2617,8 @@ DND5E.conditionTypes = {
     label: "DND5E.ConUnconscious",
     icon: "systems/dnd5e/icons/svg/statuses/unconscious.svg",
     reference: "Compendium.dnd5e.rules.JournalEntry.w7eitkpD7QQTB6j0.JournalEntryPage.UWw13ISmMxDzmwbd",
-    statuses: ["incapacitated", "prone"]
+    statuses: ["incapacitated"],
+    riders: new Set(["prone"])
```

Comment:
Let's make `riders` an array also to be consistent with `statuses`. The `DataModel` will handle coercion to a `Set`, and therefore duplicates, when the riders are applied.